scale-tests/configs/2019-09-10-testmwtdr.env

```diff
@@ -0,0 +1,158 @@
+# Depends on:
+# - TEST_NAME
+# - TEST_S3_BUCKET
+# - TEST_S3_FOLDER
+
+# Notes:
+#   mwtdr20.scaletesting.mesosphe.re
+#   - 10 public agents with 64 CPUs and 479.1 GB MEM each
+#   - 1 public agents with 64 CPUs and 479.1 GB MEM
+#   - total 704 CPUs and 5270 GB MEM
+
+# Workload configuration #######################################################
+# Total CPU quota: 34
+# Total MEM quota: 90000
+
+# CPU quota limit to set for Marathon group: 40 CPUs
+# MEM quota limit to set for Marathon group: 100000 MB MEM
+
+CLUSTER_URL="https://mwtdr20.scaletesting.mesosphe.re"
+SECURITY="strict"
+
+# Test configuration ###########################################################
+
+SHOULD_INSTALL_INFRASTRUCTURE=true
+SHOULD_INSTALL_NON_GPU_DISPATCHERS=true
+SHOULD_INSTALL_GPU_DISPATCHERS=true
+SHOULD_RUN_FAILING_STREAMING_JOBS=false
+SHOULD_RUN_FINITE_STREAMING_JOBS=true
+SHOULD_RUN_INFINITE_STREAMING_JOBS=true
+SHOULD_RUN_BATCH_JOBS=true
+SHOULD_RUN_GPU_BATCH_JOBS=false
+SHOULD_UNINSTALL_INFRASTRUCTURE_AT_THE_END=false
+
+# Infrastructure configuration #################################################
+
+GROUP_NAME="/${TEST_NAME}"
+SERVICE_NAMES_PREFIX="${TEST_NAME}/"
+INFRASTRUCTURE_OUTPUT_FILE="infrastructure.json"
+KAFKA_ZOOKEEPER_CONFIG='scale-tests/configs/kafka-zookeeper-options.json'
+KAFKA_CLUSTER_COUNT=1
+KAFKA_CONFIG='scale-tests/configs/kafka-options.json'
+CASSANDRA_CLUSTER_COUNT=1
+CASSANDRA_CONFIG='scale-tests/configs/cassandra-options.json'
+
+# Spark configuration ##########################################################
+
+# Note: leaving the Spark executor Docker image empty so that
+# executors inherit the image used for dispatchers.
+SPARK_EXECUTOR_DOCKER_IMAGE=
+
+# Service package repositories #################################################
+# Empty values will default to latest Universe packages ########################
+
+ZOOKEEPER_PACKAGE_REPO="https://universe-converter.mesosphere.com/transform?url=https://infinity-artifacts.s3.amazonaws.com/permanent/confluent-zookeeper/assets/2.6.1-5.1.2e/stub-universe-confluent-zookeeper.json"
+
+KAFKA_PACKAGE_REPO="https://universe-converter.mesosphere.com/transform?url=https://infinity-artifacts.s3.amazonaws.com/permanent/confluent-kafka/assets/2.7.1-5.3.0/stub-universe-confluent-kafka.json"
+
+CASSANDRA_PACKAGE_REPO="https://universe-converter.mesosphere.com/transform?url=https://infinity-artifacts.s3.amazonaws.com/permanent/cassandra/assets/2.7.0-3.11.4/stub-universe-cassandra.json"
+
+SPARK_PACKAGE_REPO="https://universe-converter.mesosphere.com/transform?url=https://infinity-artifacts-ci.s3.amazonaws.com/autodelete7d/spark/20190912-024512-ET24v7j1oppCgD0s/stub-universe-spark.json"
```

Comment:
Should we delete? the `autodelete` url might cause issues in the future for test runners.

Reply:
Good point. Addressed it here: https://github.com/mesosphere/spark-build/pull/551/commits/a353d4003b29b447581429938d5533daf8a4f6f9